tests/x11/yast2_users.pm

```diff
@@ -1,29 +0,0 @@
-# SUSE's openQA tests
-#
-# Copyright © 2009-2013 Bernhard M. Wiedemann
-# Copyright © 2012-2016 SUSE LLC
-#
-# Copying and distribution of this file, with or without modification,
-# are permitted in any medium without royalty provided the copyright
-# notice and this notice are preserved.  This file is offered as-is,
-# without any warranty.
-
-# Summary: Test yast2 users configuration module
-# Maintainer: Oliver Kurz <okurz@suse.de>
-
-use base "x11test";
-use strict;
-use testapi;
-
-sub run {
-    x11_start_program("xdg-su -c '/sbin/yast2 users'");
-    if ($password) {
-        type_password;
-        send_key "ret";
-    }
-    assert_screen 'test-yast2_users-1', 60;
-    send_key "alt-o";    # OK => Exit
-}
-
-1;
-# vim: set sw=4 et:
```

Comment:
Why removing this test module?

Reply:
because it's unused. I created https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3683 to make the removal more explicit.

Reply:
I've commented it in that PR. I see usages of this test module, hence we need either do more clean up, or leave it =)